### DISCUSSION TITLE
Module not found: Can't resolve 'devlop

### DISCUSSION THREAD

**Discussion by BabyMing** [unifiedjs/unified#233] [Q&A] [answered]
In react Module not found: Can't resolve 'devlop' in 'E:\code\asset-management-portal-master\asset-management-ui\.yarn\cache\unified-npm-11.0.3-6e4da25f5f-402d6b397b.zip\node_modules\unified\lib'
how  can i do resolve it? I really need help

**Comment by remcohaszing** [comment, marked as answer]
It looks like you’re using an outdated bundler (one that doesn’t support [package exports](https://nodejs.org/api/packages.html#exports)). You should probably upgrade to something more modern.

**Reply by BabyMing** [reply]
react is 17.0.2
codemirror is ^6.0.1
"unified" is  "^11.0.3",
but is still report the error

./.yarn/cache/mdast-util-to-hast-npm-13.0.2-87e682f4f1-8fef6c3752.zip/node_modules/mdast-util-to-hast/lib/index.js
Module not found: Can't resolve 'devlop' in 'E:\code\asset-management-portal-master\asset-management-ui\.yarn\cache\mdast-util-to-hast-npm-13.0.2-87e682f4f1-8fef6c3752.zip\node_modules\mdast-util-to-hast\lib'

I  dont really know it

**Reply by wooorm** [reply]
Update your bundler. Bundlers are for example esbuild, webpack, rollup, etc.
And make sure to not use Jest.
